app/models/metric/ci_mixin/capture.rb

```diff
@@ -16,7 +16,7 @@ def queue_name_for_metrics_collection
             raise "Unsupported type #{self.class.name} (id: #{id})"
           end
 
-    ems.class.name[3..-1].underscore
+    ems.class.metrics_collect_queue_name
```

Comment:
you may want to have EMS have an instance method that calls the class method, so that the caller has `ems.metrics_collect_queue_name`